README.md

```diff
@@ -26,13 +26,13 @@ on how to install it for your system.
 To install this package, execute the following command inside Julia's REPL:
 
 ```julia
-] add IntervalMatrices
+pkg> add IntervalMatrices
```

Comment:
if someone is new to julia can be confused, maybe keep the current version or write:

```julia
julia> ]
pkg> add IntervalMatrices
```

Reply:
I would argue that seeing the bracket can be confusing as well. Should we just copy the same section from [here](https://github.com/JuliaReach/LazySets.jl#installing)?

Reply:
> Should we just copy the same section from here?

Yes, that is better.